lib/application/index.js

```diff
@@ -4,6 +4,9 @@
  * configuration and rendering.
  */
 
+// Local
+var Promise = require('../polyfills/promise');
```

Comment:
The scoutfile is polyfilling it already, itself... All this would do is shift those bytes from that project to this one.